bot/admin/admin.py

```diff
@@ -316,6 +414,50 @@ def _create_cogs_embed_string(loaded_cogs: Mapping[str, commands.Cog]) -> str:
     return string
 
 
+def _build_botonly_embed(is_enabled_string: str):
+    """Creates an embed for the botonly command.
+
+    Args:
+        is_enabled_string (str): A string which will be interpolated into the title. Should contain the word 'aktiviert'
+        or 'deaktiviert'.
+
+    Returns:
+        (discord.Embed): An embed containing information, if the botonly feature was en- or disabled for a channel.
+    """
+    title = 'Bot-Only Funktionalität wurde für diesen Channel {0}'.format(is_enabled_string)
+    description = 'Ein Bot-Only Channel ist ein Channel in dem nur ein Bot Nachrichten posten kann. Jede Nachricht von' \
```

Comment:
Der Bot-only Mode sorgt dafür, dass nur noch SAM Nachrichten in einem Channel posten darf. Jede ....